externals/kyuubi-spark-sql-engine/src/main/scala/org/apache/spark/kyuubi/SQLOperationListener.scala

```diff
@@ -61,6 +63,9 @@ class SQLOperationListener(
       None
     }
 
+  val operationRunTime = new AtomicLong(0)
+  val operationCpuTime = new AtomicLong(0)
```

Comment:
External access variables should be allowed to read only.

```suggestion
  private val operationRunTime = new AtomicLong(0)
  private val operationCpuTime = new AtomicLong(0)

  def getOperationRunTime: Long = operationRunTime.get()

  def getOperationCpuTime: Long = operationCpuTime.get()
```